Sources/Fluent/Query/Filter.swift

```diff
@@ -129,3 +132,15 @@ extension QueryRepresentable {
         return try filter(T.self, field, .contains, value)
     }
 }
+
+extension QueryRepresentable {
+    @discardableResult
+    public func raw(command: String, values: [NodeRepresentable] = []) throws -> Query<Self.T> {
```

Comment:
We'll need to check with @Casperhr to make sure this completely solves what he's wanting.

I know in Laravel there was strange things like:

```php
User::where('timestamp', '>', DB::raw("TIMESTAMP()"))->all()
```

This basically allows you to bypass the ` escape that happens to strings that are passed.

I don't know if that's some we want or need or if this level of granularity is fine.